torch/fx/proxy.py

```diff
@@ -126,7 +126,10 @@ def create_node(self, kind : str, target : Target,
             self.scope.module_path,
             self.scope.module_type,
         )
-        if self.module_stack:
+        current_meta: Dict[str, Tuple[str, Any]] = fx_traceback.get_current_meta()
```

Comment:
need to guard this path with

if fx_traceback.has_preserved_node_meta():

Reply:
Sure!